### ISSUE TITLE
Control tab returns error

### ISSUE THREAD

**Issue by Doetheman** [GhostWalker562/flutterbook#8]
![Screen Shot 2021-11-29 at 12 13 10 PM (2)](https://user-images.githubusercontent.com/23367376/143916070-764e61dd-27f7-4fc1-8059-94b88403172f.png)

Also ran the code from the example directory of this repository and it returned the same error. 

Flutter Doctor
[✓] Flutter (Channel stable, 2.5.3, on macOS 11.5.2 20G95 darwin-x64, locale en-US)
[✗] Android toolchain - develop for Android devices
    ✗ Unable to locate Android SDK.
      Install Android Studio from: https://developer.android.com/studio/index.html
      On first launch it will assist you in installing the Android SDK components.
      (or visit https://flutter.dev/docs/get-started/install/macos#android-setup for detailed instructions).
      If the Android SDK has been installed to a custom location, please use
      `flutter config --android-sdk` to update to that location.

[✓] Xcode - develop for iOS and macOS
[✓] Chrome - develop for the web
[✓] Android Studio (version 2020.3)
[✓] VS Code (version 1.62.3)
[✓] Connected device (1 available)

**Comment by dbbd59** [comment]
hi! @Doetheman 
you can use my fork for now:

```
  flutterbook:
    git:
      url: https://github.com/dbbd59/flutterbook.git
      ref: bugfix/controls_bottom_bar_provider_issue
```
I've also added this https://github.com/GhostWalker562/flutterbook/pull/7 pull request:

Bye!

**Comment by GhostWalker562** [comment]
I'll get a new release in by the end of the week. This should be fixed when I merge the PR!

**Comment by GhostWalker562** [comment]
Going to release when I get in the docs feature :)

**Comment by Josiassejod1** [comment]
This got addressed in the release #12
